### ISSUE TITLE
A11y: Dropdown component have more than one tabbale element

### ISSUE THREAD

**Issue by icemanlily** [IBM/carbon-components-react#2081]
The issue comes from #1851 .

## Environment

> Operating system
MacOS

> Browser
Chrome

> Automated testing tool and ruleset
Manual

## Detailed description

> What version of the Carbon Design System are you using?
6.58.0

> What did you expect to happen?
Dropdown list should be focused only one time to operate.

> What happened instead?
We must type tab key twice to focus on the dropdown list. When we focus on the dropdown list first time, we can't operate it. Then type the tab key again, We can see the focus is not changed and we can type space to operate the dropdown.

> What WCAG 2.1 checkpoint does the issue violate?
2.1.1 Keyboard

## Steps to reproduce the issue

1. Go to https://dev.console.test.cloud.ibm.com/login
2. Type tab key to focus on the Dropdown list.

## Additional information
![image](https://user-images.githubusercontent.com/6970471/55052424-d1b18f00-5093-11e9-8edc-74bbbaeeeafb.png)

![image](https://user-images.githubusercontent.com/6970471/55052561-65835b00-5094-11e9-8a09-44e464d2d708.png)

**Comment by shinytoyrobots** [comment]
Can you please update the issue with specific information and reproduction steps as outlined in the Accessibility Issue template? Thank.



## Environment

> Operating system

> Browser

> Automated testing tool and ruleset

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?

> What did you expect to happen?

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox

## Additional information

- Screenshots or code
- Notes
